packages/support/src/support-server.ts

```diff
@@ -211,7 +211,7 @@ export class SupportServer {
         return;
       }
 
-      next();
+      return next();
```

Comment:
I think calling `return` will result in to subsequent middlewares not being called. But I'm not 100% sure.

Reply:
it's fine as long as `next` is called